frontend/src/admin/components/sidebar/sidebar.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import UserList from "./userList/userList";
+import RestartServerButton from "./restart/restartServerButton";
+import "./sidebar.scss";
+
+export default function Sidebar() {
+    return (
+        <section className={"c-sidebar"}>
```

Comment:
Curly brackets are unnecessary. You have no dynamic content in the className string.
```suggestion
        <section className="c-sidebar">
```

Reply:
✅